homeassistant/components/nfandroidtv/manifest.json

```diff
@@ -5,6 +5,7 @@
   "requirements": ["notifications-android-tv==0.1.5"],
   "codeowners": ["@tkdrob"],
   "config_flow": true,
+  "dhcp": [{ "hostname": "amazon-*" }],
```

Comment:
I don't think this PR is a good idea because it relies on us  suggesting an integration that relies on a third party app